CMake/ctkMacroBuildPlugin.cmake

```diff
@@ -371,7 +371,7 @@ macro(ctkMacroBuildPlugin)
     PREFIX "lib"
     )
 
-  if(NOT MY_TEST_PLUGIN AND NOT MY_NO_INSTALL)
+  if(NOT MY_TEST_PLUGIN AND NOT MY_NO_INSTALL AND MY_LIBRARY_TYPE STREQUAL "SHARED")
```

Comment:
* The install rule should still be active for static build. Indeed, the `Development` is relevant in that case.
* Since `LIBRARY_TYPE` is not listed as an argument of `ctkMacroBuildPlugin` (see above), this variable is not relevant